### PR TITLE
feat(mods/pride_flags): autism pride flag

### PR DESCRIPTION
**Pull request by AlecWhite**
## Purpose of change
Added the autism pride flag.

### CODE REVIEW

File: data/mods/pride_flags/flags.json [hunk 1]
```diff
@@ -86,5 +86,13 @@
     "name": { "str": "Genderfluid pride flag" },
     "description": "A genderfluid pride flag.  The pink represents feminimity, the white represents lack of gender, the purple represents androgyny, the black represents all other genders, and the blue represents masculinity.",
     "color": "blue"
+  },
+  {
+    "id": "autism_flag",
+    "type": "ARMOR",
+    "copy-from": "american_flag",
+    "name": { "str": "Autism pride flag" },
+    "description": "The autistic pride flag. Gold middle band refers to the elemental symbol of gold (AU) which is used in the autistic community. The gradient refers to the Pride Movement. Red originated as a protest to the `light it up blue` campaign, which was denounced by the autistic community. In color theory, green is the complimentary color to red. The infinity sign is a symbol long used in the wider autistic and wider neurodiverse community.",
```
Comment (RobbieNeko) [inline data/mods/pride_flags/flags.json:95]:
```suggestion
    "description": "The autistic pride flag.  Gold middle band refers to the elemental symbol of gold (AU) which is used in the autistic community.  The gradient refers to the Pride Movement. Red originated as a protest to the `light it up blue` campaign, which was denounced by the autistic community.  In color theory, green is the complimentary color to red.  The infinity sign is a symbol long used in the wider autistic and wider neurodiverse community.",
```
Here, hopefully this'll fix it being annoyed about your spacing

**Comment by RobbieNeko** [comment]
Aside from the tests being annoyed by single-spacing as usual, the additions look good to me.

**Review by chaosvolt — DISMISSED** [review]
Fucking semantic PR titles weh

File: data/mods/pride_flags/flags.json [hunk 2]
```diff
@@ -86,5 +86,13 @@
     "name": { "str": "Genderfluid pride flag" },
     "description": "A genderfluid pride flag.  The pink represents feminimity, the white represents lack of gender, the purple represents androgyny, the black represents all other genders, and the blue represents masculinity.",
     "color": "blue"
+  },
+  {
+    "id": "autism_flag",
+    "type": "ARMOR",
+    "copy-from": "american_flag",
+    "name": { "str": "Autism pride flag" },
+    "description": "The autistic pride flag.  Gold middle band refers to the elemental symbol of gold (AU) which is used in the autistic community.  The gradient refers to the Pride Movement. Red originated as a protest to the `light it up blue` campaign, which was denounced by the autistic community.  In color theory, green is the complimentary color to red.  The infinity sign is a symbol long used in the wider autistic and wider neurodiverse community.",
```
Comment (chaosvolt) [inline data/mods/pride_flags/flags.json:95]:
```suggestion
    "description": "The autistic pride flag.  Gold middle band refers to the elemental symbol of gold (AU) which is used in the autistic community.  The gradient refers to the Pride Movement.  Red originated as a protest to the `light it up blue` campaign, which was denounced by the autistic community.  In color theory, green is the complimentary color to red.  The infinity sign is a symbol long used in the wider autistic and wider neurodiverse community.",
```

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**